diff --git a/dist/provider/external/getParaCosmosLib.js b/dist/provider/external/getParaCosmosLib.js
index e0e92fdbfdd8f5a01423ee054cdd5fac6d6a387a..6c3b71fb6ebd0701d9858a5a0796aa58f509b83a 100644
--- a/dist/provider/external/getParaCosmosLib.js
+++ b/dist/provider/external/getParaCosmosLib.js
@@ -5,7 +5,7 @@ import {
 const getParaCosmosLib = () => __async(void 0, null, function* () {
   let lib;
   try {
-    lib = yield import("@getpara/cosmos-wallet-connectors");
+    lib = void 0;
   } catch (e) {
     lib = void 0;
   }
diff --git a/dist/provider/external/getParaSolanaLib.js b/dist/provider/external/getParaSolanaLib.js
index a9fba1a5ae4636f4a01dee7e9e270a8915e6a80d..c6e784caf42a7c347670ccec7d079bc8772b6ee0 100644
--- a/dist/provider/external/getParaSolanaLib.js
+++ b/dist/provider/external/getParaSolanaLib.js
@@ -5,7 +5,7 @@ import {
 const getParaSolanaLib = () => __async(void 0, null, function* () {
   let lib;
   try {
-    lib = yield import("@getpara/solana-wallet-connectors");
+    lib = void 0;
   } catch (e) {
     lib = void 0;
   }
